doc/03_argotcli.md

```diff
@@ -421,6 +421,8 @@ The user can additionally provide the following flags:
 
 `summarize` builds the dataflow summaries of the functions matching the regex provided as argument, or all the functions that should be summarized according to the configuration if no argument is provided. For more information about the summarization, refer to the guide for the [taint analysis tool](01_taint.md#taint-analysis).
 
+> Summarize alone when the dataflow-problems options summarize-on-demand is true does not run the intra-procedural analysis, it only builds the data-structure necessary to summarize the function later. 
```

Comment:
nit: this sentence is a bit confusing. Maybe something like "when the `dataflow-problems` option `summarize-on-demand` is true, the `summarize` command does not run the intra-procedural analysis to summarize the function, but instead enables the function to be automatically summarized later when needed"